src/core/Tile/Tile.tsx

```diff
@@ -193,15 +195,15 @@ export const Tile = (props: TileProps) => {
               aria-hidden
             />
           )}
-          <span className='iui-name-label'>{name}</span>
+          <span className='iui-tile-name-label'>{name}</span>
         </div>
 
         {description != undefined && (
-          <div className='iui-description'>{description}</div>
+          <div className='iui-tile-description'>{description}</div>
         )}
 
         {metadata != undefined && (
-          <div className='iui-metadata'>{metadata}</div>
+          <div className='iui-tile-metadata'>{metadata}</div>
```

Comment:
Just noticed, we dont user `iui-tile-metadata-icon` anywhere :O

Reply:
I could not find it in the CSS either

Reply:
![image](https://user-images.githubusercontent.com/81580355/152541588-5915a11d-e614-4d34-88bd-1f8bda7278d8.png)

Reply:
I think it may have been added for webcomponents? Not sure. But I see the CSS also targets the `> svg` so there is no class needed in React

Reply:
weird.. As long as svg is there, its fine, but it might be also img, then its gonna be broken